ruby/spec/cucumber/ci_environment/variable_expression_evaluate_spec.rb

```diff
@@ -1,38 +1,40 @@
+# frozen_string_literal: true
+
 require 'cucumber/ci_environment'
 
-describe 'Cucumber::CiEnvironment::VariableExpression.evaluate' do
-  include Cucumber::CiEnvironment::VariableExpression
```

Comment:
No need to amend this as this isn't the way to write these kinds of specs, we should be using isolated lets. But given this is already here we can ignore this and fix it later.

Reply:
I am curious as to how the let refactor would work and what is the difference? From looking at this spec file I can see what it is testing, the inputs and expected outputs are readily seen. What is bad about this rspec file?

Reply:
When you `include` something inside a describe block it pollutes the whole namespace. Dependent on how you ran / checked rubocop it will flag this to you. A better way it to use an isolated anonymous class inside a let so something like

```
let(:foo) do
  Class.new(inheritance) do
    include RequiredModule1
    include RequiredModule2
  end
end
```

Now the inclusion is completely isolated inside the memoized let.